bin/spectatord_main.cc

```diff
@@ -72,6 +72,7 @@ ABSL_FLAG(bool, debug, false,
           "Debug spectatord. All values will be sent to a dev aggregator and "
           "dropped.");
 ABSL_FLAG(bool, verbose, false, "Use verbose logging");
+ABSL_FLAG(bool, trace, false, "Trace http requests");
```

Comment:
Minor comment, but trace makes me think of things like zipkin tracing rather than curl's verbose logging.

Reply:
Valid point. It did cross my mind as I was typing the option while testing it. I'll rename it.